packages/language-server/test/snapshots/xml-view-diagnostics/non-stable-id/output-lsp-response.json

```diff
@@ -17,6 +17,6 @@
     "severity": 1,
     "source": "UI5 Language Assistant",
     "message": "The class \"Dialog\" must declare a non-empty ID attribute when flexEnabled is \"true\"",
```

Comment:
"The \"Dialog\"class can't have an empty ID attribute when flexEnabled is \"true\".